flash/text/classification/data.py

```diff
@@ -330,6 +373,7 @@ def __init__(
                 DefaultDataSources.CSV: TextCSVDataSource(self.backbone, max_length=max_length),
                 DefaultDataSources.JSON: TextJSONDataSource(self.backbone, max_length=max_length),
                 "data_frame": TextDataFrameDataSource(self.backbone, max_length=max_length),
+                "list": TextListDataSource(self.backbone, max_length=max_length),
```

Comment:
Let's create a List within DefaultDataSources enum :) Same for DataFrame and Sentences